geomesa-hbase/geomesa-hbase-server-hbase1/src/main/scala/org/locationtech/geomesa/hbase/coprocessor/GeoMesaCoprocessor.scala

```diff
@@ -0,0 +1,15 @@
+/***********************************************************************
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.hbase.coprocessor
+
+/**
+ * Kept around for compatibility with existing tables configured with geomesa 2.4 or earlier
+ */
+@deprecated("Replaced with org.locationtech.geomesa.hbase.server.coprocessor.GeoMesaCoprocessor")
```

Comment:
Did 2.4.0 ship with the absolute timeout?

If so, there's no compatibility between 3.x and earlier than 2.4.  Being compatible with 3.0 is a nice-to-have.

Reply:
If part of the package shuffling is heading toward Java 11 support, let's break things early and apologize in the HBase part of the upgrade guide.

Reply:
ah, well you can upgrade the distributed-runtime jar without updating the table config i think, depending on how you installed it. if we drop this class then we would force a table config change. but i'm not sure if it would break java 11 or not, since it would only be in the distributed-runtime uber jar...